pyproject.toml

```diff
@@ -1,8 +1,25 @@
 [tool.mypy]
-warn_unused_configs = true
 python_version = "3.10"
-mypy_path = "src/py"
+mypy_path = ["src/py", "pyodide-build"]
+show_error_codes = true
+warn_unreachable = true
+enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
+
+# Strict checks
```

Comment:
Ideally, all of these should eventually be true (and then you could just replace them with `strict = true`), but that would take some work. I tried to enable a few easy ones.